src/core/modules/Planet.hpp

```diff
@@ -222,6 +222,7 @@ class Planet : public StelObject
 	virtual double getAngularSize(const StelCore* core) const Q_DECL_OVERRIDE;
 	virtual bool hasAtmosphere(void) {return atmosphere;}
 	virtual bool hasHalo(void) {return halo;}
+	bool hasValidData();
```

Comment:
Please document what "valid" means in this context! Maybe rename to hasValidOrbitData() or hasValidPositionalData()

Reply:
Good suggestion! 

Done.